src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>AngularPractice</title>
+  <title>RoutingServiceDemo</title>
```

Comment:
The title can be `User Management System` which is as per the task

Reply:
changed the title to user management system